docs/dev_guide/ceph_s3_tests_guide.md

```diff
@@ -0,0 +1,143 @@
+```bash
+nb system set-debug-level <level>
+```
+A good level to start with is 2 or 3. the higher you go the more verbose and noisy the logs will become.
+
+## 3) Deploy The Tester Deployment 
+Use the `test_ceph_s3_deployment.yml` file to install the tester pod. This will install a pod that does nothing, but it allows us to go into the pod and run our test
+```bash
+kubectl apply -f src/test/system_tests/ceph_s3_tests/test_ceph_s3_deployment.yml
+```
+
+## 4) Setup Test Config (Inside The Tester Pod)
+Once the tester pod is up, we can go into it and prepare the environment to run the tests.
+both `kubectl` and `oc` can be used to go into the pod:
+```bash
+# if you have kubectl 
+kubectl exec -it [noobaa-tester pod] -- bash
+# or with oc
+oc rsh [noobaa-tester pod] bash
+```
+
+In the tester pod, go to noobaa working directory:
+```bash
+cd /root/node_modules/noobaa-core/
+```
+
+Run the script that will create the necessary accounts in noobaa and update the Ceph tests config file accordingly:
+```bash
+node ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_config_setup.js
+```
+
+Note: If you want to ignore PythonDeprecationWarnings use (which will then ignore all Python warnings, so keep that in mind):
+```bash
+export PYTHONWARNINGS="ignore"
+```
+
+### 5) Run a Test (Inside The Tester Pod)
+To run a test, from noobaa working directory:
+```bash
+S3TEST_CONF=src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf ./src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/bin/nosetests <test name>
+```
+This should run the test on the noobaa deployment we've set up.
+
+### 6) Debug a Test (Inside The Tester Pod)
+You can view the test (python) by going to the test file and searching for the test function. e.g. if you are working on test `s3tests_boto3.functional.test_s3:test_set_bucket_tagging` then you should `vim ./src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py` and search for the function `test_set_bucket_tagging`.  
+
+The best place to start investigating is noobaa endpoint logs. if you are running with debug level that is higher than 1, you should see messages of the S3 requests with the prefix `S3 REQUEST`. S3 replies will be with the prefix `HTTP REPLY`.
+
+### 7) Change a Test
+If there is a test that you suspect that it has a faulty and you would like to change the code, you will need to direct to your own forked repo. Since the file `./src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py` is a read-only file, floow the next steps:
+
+1) Fork and clone the repository ceph/s3-tests.
+2) Create a new branch from the hash number that was set in the file `./src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh`.
+3) Change the code, commit and push to remote branch.
+4) Change the values of `CEPH_LINK` to your remote repo and the `CEPH_TESTS_VERSION` to the newest commit.
+5) Build tester image again, deploy noobaa and run the test.
+
+# Run All Ceph S3 Tests
```

Comment:
also, change here to be a sub-header and not a main header
```suggestion
## Run All Ceph S3 Tests
```
also fix all the steps below from 2 hashes to 3